tests/article/article.js

```diff
@@ -105,4 +105,71 @@ describe('Tests for article resource', () => {
       expect(body).to.not.have.property('data');
     });
   });
+
+  describe('Tests for liking/disliking Articles', () => {
+    let token;
+    let articleId;
+    before(async () => {
+      const { body: { data: { token: userToken } } } = await chai.request(app)
+        .post('/api/users/login')
+        .send(user1.logIn);
+
+      token = userToken;
+
+      const { body: { data: { id } } } = await chai.request(app)
+        .post('/api/articles')
+        .set('Authorization', `Bearer ${userToken}`)
+        .send(mockArticle);
+
+      articleId = id;
+    });
+
+    it('Should return unauthorized when no token is provided', async () => {
+      const res = await chai.request(app)
+        .patch(`/api/articles/${articleId}/likes`);
+      expect(res).to.have.status(401);
+    });
+
+    it('Should return unauthorized when no token is provided', async () => {
+      const res = await chai.request(app)
+        .patch(`/api/articles/${articleId}/dislikes`);
+      expect(res).to.have.status(401);
+    });
+
+    it('Should create an impression liking an article ', async () => {
+      const res = await chai.request(app)
+        .patch(`/api/articles/${articleId}/likes`)
+        .set('Authorization', `Bearer ${token}`);
+      const { body } = res;
+      expect(res).to.have.status(200);
+      expect(body).to.have.property('data');
+    });
+
+    it('Should update an impression unliking an article ', async () => {
+      const res = await chai.request(app)
+        .patch(`/api/articles/${articleId}/likes`)
+        .set('Authorization', `Bearer ${token}`);
+      const { body } = res;
+      expect(res).to.have.status(200);
+      expect(body).to.have.property('data');
+    });
+
+    it('Should update an impression disliking an article ', async () => {
+      const res = await chai.request(app)
+        .patch(`/api/articles/${articleId}/dislikes`)
+        .set('Authorization', `Bearer ${token}`);
+      const { body } = res;
+      expect(res).to.have.status(200);
+      expect(body).to.have.property('data');
```

Comment:
@Ibidapo Please apply the above comment to fit the required outcome for the remaining test cases.